<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>esparza_window</class>
 <widget class="QDialog" name="esparza_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1395</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_insertar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_dato">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_fila">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_columna">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dato</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_picture">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>1001</width>
     <height>701</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Funciones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DeleteCell">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Cell</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="delete_cell_fil">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="delete_cell_col">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_max_row">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Max_Row</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_max_row_fila">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_max_col">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Max_Col</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_max_col_col">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_min_row">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Min_Row</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_min_row_fila">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_min_col">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Min_Col</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_min_col_col">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sort_row">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort_Row</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_sort_row_fila">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sort_col">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort_Col</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_sort_col_col">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete_row">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete_Row</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_delete_row_row">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Fila</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete_col">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete_Col</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_delete_col_col">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Columna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mix">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_transpuesta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Transpuesta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
